engine/src/main/java/org/apache/apex/stram/ContainerRedeployRequest.java

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
I remember discussing a generic deployment grouping concept that covers initial deployment as well (and not just redeploys). Based on that should we change the name to ContainerDeployRequest?